recipes-kernel/linux-firmware-qca6564/linux-firmware-qca6564_2.4-r2.2.bb

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2019 Digi International Inc.
+
+SUMMARY = "Firmware files for Qualcomm's QCA6564 wireless chip"
+SECTION = "base"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://LICENCE.atheros_firmware;md5=30a14c7823beedac9fa39c64fdd01a13"
+
+SRC_URI = " \
+    file://nvm_tlv_3.2.bin \
+    file://rampatch_tlv_3.2.tlv \
+    file://bdwlan30_US.bin \
+    file://LICENCE.atheros_firmware \
+    file://otp30.bin \
+    file://qwlan30.bin \
+    file://utf30.bin \
+"
```

Comment:
I am fine in you adding the recipe however we won't commit the binaries on the layer. Check if NXP does not has a tarball or Git for source, or put it on your github so we avoid putting binaries here.

Reply:
I will add them to a compressed tarball and push the changes.